test/TestApps/AppLifecycleTestApp/Helpers.cpp

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#include "pch.h"
+#include "Helpers.h"
+#include <TestDef.h>
+
+const wchar_t* g_bootStrapDllName = L"Microsoft.ProjectReunion.Bootstrap.dll";
+
+typedef HRESULT (*BootStrapTestInit)(PCWSTR prefix, PCWSTR publisherId);
+typedef HRESULT (*BootStrapInit)(const UINT32 majorMinorVersion, const PACKAGE_VERSION minVersion);
+typedef void (*BootStrapShutdown)();
+
+bool IsPackagedProcess()
+{
+    UINT32 n{};
+    return ::GetCurrentPackageFullName(&n, nullptr) == ERROR_INSUFFICIENT_BUFFER;;
+}
```

Comment:
Dupe(ish) of IsPackagedProcess() in test\inc\ProjectReunion.Test.AppModel.h

Remove, #include <ProjectReunion.Test.AppModel.h> and add namespace prefix to callers (or "using Test::AppModel;" ?)